src/cocaine-app/jobs/ttl_cleanup.py

```diff
@@ -104,3 +104,31 @@ def _involved_couples(self):
     def on_complete(self, processor):
         couple = str(storage.groups[self.iter_group].couple)
         processor.planner.update_cleanup_ts(couple, time.time())
+
+    @staticmethod
+    def report_resources(params):
+        """
+        Report resources supposed usage for specified params
+        :param params: params to be passed on creating the job instance
+        :return: dict={'groups':[], 'resources':{ Job.RESOURCE_HOST_IN: [], etc}}
+        """
+
+        # XXX: this code duplicates 'set_resources', 'involved_groups' methods but this duplication is chose
+        # to minimize changes to test
+        res = {}
+
+        iter_group = params['iter_group']
+
+        if iter_group not in storage.groups:
+            raise ValueError("Invalid params {}".format(iter_group))
```

Comment:
fixed in https://github.com/yandex/mastermind/pull/434/commits/968d0fbf32e9a25f5713cb1d2e5c4d336cf6a401